source/whats-new.txt

```diff
@@ -33,6 +41,30 @@ Learn what's new in:
 * :ref:`Version 1.1 <version-1.1>`
 * :ref:`Version 1.0 <version-1.0>`
 
+.. _version-1.13:
+
+What's New in 1.13
+------------------
+
+The 1.13 {+driver-short+} release includes the following improvements and fixes:
+
+- Logging for server selection and SDAM. To learn more about logging, see the
+  :ref:`golang-logging` documentation for more information about logging.
```

Comment:
S: I think there are too many clauses in sentence 2

Reply:
doh! good catch